### ISSUE TITLE
set the directory where you work

### ISSUE THREAD

**Issue by N3koSempai** [massivemadness/Squircle-CE#62]
when working on a file in a folder if you closed 
the program and reopen the ide returns to the root directory instead of saving the position of the last open file

**Comment by Ilithy** [comment]
Indeed, it would save time, I would like to see this also happen in Squircle IDE

**Comment by nomen140** [comment]
1+

**Comment by massivemadness** [comment]
Hello @N3koSempai, I implemented this in [v2025.1.0](https://github.com/massivemadness/Squircle-CE/releases/tag/2025.1.0)
